pkg/controller/elasticsearch/remotecluster/annotations.go

```diff
@@ -5,58 +5,42 @@
 package remotecluster
 
 import (
-	"encoding/json"
+	"sort"
+	"strings"
 
 	esv1 "github.com/elastic/cloud-on-k8s/pkg/apis/elasticsearch/v1"
 	"github.com/elastic/cloud-on-k8s/pkg/utils/k8s"
-	"github.com/pkg/errors"
 )
 
 const (
-	RemoteClustersAnnotationName = "elasticsearch.k8s.elastic.co/remote-clusters"
+	// CreatedRemoteClustersAnnotationName holds the list of the remote clusters which have been created
+	CreatedRemoteClustersAnnotationName = "elasticsearch.k8s.elastic.co/managed-remote-clusters"
```

Comment:
`CreatedRemoteClustersAnnotationName ` -> `ManagedRemoteClustersAnnotationName ` to better match the annotation name?